src/components/Tabs/KitTabPanels.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="kit-tab-panels" ref="container" :style="style">
+    <slot />
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+export default Vue.extend({
+  name: 'KitTabPanels',
+  inject: {
+    state: {
+      default: () => ({
+        activeTab: undefined
+      })
+    }
+  },
+  data() {
+    return {
+      style: { maxHeight: 'inherit' },
+      timer: null
+    }
+  },
+  watch: {
+    'state.activeTab'() {
```

Comment:
not possible to avoid this kind of deep watch by replacing the entire state when updated ? (may be constraint of provide/inject ?)

Reply:
Don't thinks so.